dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -78,6 +89,16 @@ public ItemRest findOne(Context context, UUID id) {
         return page;
     }
 
+    @Override
+    public void patch(Context context, HttpServletRequest request, String apiCategory, String model, UUID uuid, Patch
+            patch)
+            throws UnprocessableEntityException, PatchBadRequestException, SQLException, AuthorizeException {
+
```

Comment:
This method should be responsible to retrieve the item so to centralize the management of a 404 error and reduce code duplication.